swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
```

Comment:
This is the perfect time to use a [ternary operator](https://docs.python.org/3/reference/expressions.html#conditional-expressions).
```suggestion
        self.id = id if id is not None else uuid.uuid4().int
```